WordPress/Classes/Utility/Blogging Reminders/BloggingRemindersStore.swift

```diff
@@ -0,0 +1,306 @@
+import Combine
+
+/// Store for the blogging reminders.  This class should not be used directly.  Please refer to `BloggingReminders` instead.
+///
+class BloggingRemindersStore {
```

Comment:
Totally optional, but this could also be a protocol and we could for example have different concrete implementations if, for example, in the future we decide to use a different type of persistence.

Reply:
Do you mind if I leave it as-is for now.  I don't mind if we iterate right after to make it more flexible - I'm just not sure it's a good idea to try to go this route for the initial implementation of the solution purely from a timing perspective.

Reply:
Do you think we could leave this as something to tackle iteratively?

I think this is something we can discuss and implement iteratively to keep the PR simpler for v1.

Reply:
sounds good to me